homeassistant/components/gios/air_quality.py

```diff
@@ -0,0 +1,157 @@
+"""Support for the GIOS service."""
+from homeassistant.components.air_quality import (
+    ATTR_CO,
+    ATTR_NO2,
+    ATTR_OZONE,
+    ATTR_PM_2_5,
+    ATTR_PM_10,
+    ATTR_SO2,
+    AirQualityEntity,
+)
+from homeassistant.const import CONF_NAME
+
+from .const import ATTR_STATION, DATA_CLIENT, DOMAIN, ICONS_MAP
+
+ATTRIBUTION = "Data provided by GIOŚ"
+
```

Comment:
Should we set the default `SCAN_INTERVAL` to 30 minutes here to not try to update more often than the throttle?

Reply:
Of course you're right :)